x/wasm/internal/keeper/bench_test.go

```diff
@@ -0,0 +1,55 @@
+package keeper
+
+import (
+	"github.com/CosmWasm/wasmd/x/wasm/internal/types"
+	"github.com/stretchr/testify/require"
+	"github.com/syndtr/goleveldb/leveldb/opt"
+	dbm "github.com/tendermint/tm-db"
+	"testing"
+)
+
+func BenchmarkExecution(b *testing.B) {
```

Comment:
I'd love to see the numbers from this on, say, wasmd 0.12 (which had the old wasmer engine), so we can compare the speedup for uncached over the last 3-4 months, plus the huge speedup for pinned contracts